scripts/feature_tests/README.md

```diff
@@ -95,16 +124,24 @@ export CAPM3RELEASE=v0.3.2
 export CAPIRELEASE=v0.3.4
```

Comment:
v0.3.4 has a bug (https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/805#issuecomment-661298583) that leads to metal3-dev-env 'make' failure,
so please update:
export CAPIRELEASE=v0.3.5

Reply:
@Jaakko-Os I have not updated anything related to the upgrade part since I was not sure about which parts are completed which not. I can update vars but then documentation will still miss for example trigger phrase for centos upgrade, build status for centos upgrade, etc. Do you want me cover centos upgrade part as well, can we already note that information here?

Reply:
You're right. Better to do the upgrade updates once the CI part is merged. I'll take that as an action point.